openedx/core/djangoapps/programs/utils.py

```diff
@@ -10,7 +10,7 @@
 from opaque_keys.edx.keys import CourseKey
 import pytz
 
-from lms.djangoapps.certificates.api import get_certificates_for_user, is_passing_status
+from lms.djangoapps.certificates import api as certificate_api
```

Comment:
Nit: I would have probably called this 'certificates_api' if I was the one making the changes.